birdhouse/README.rst

```diff
@@ -150,11 +138,11 @@ https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests with pre-configured
 Jenkins at https://github.com/Ouranosinc/jenkins-config.
 
 For that test suite to pass, run the script
-`\ ``bootstrap-instance-for-testsuite`` <scripts/bootstrap-instance-for-testsuite>`_
+`bootstrap-instance-for-testsuite <birdhouse/scripts/bootstrap-instance-for-testsuite>`_
 to prepare your new instance.  Further documentation inside the script.
 
 Optional component
-`all-public-access <optional-components#give-public-access-to-all-resources-for-testing-purposes>`_
+`all-public-access <birdhouse/optional-components#give-public-access-to-all-resources-for-testing-purposes>`_
```

Comment:
This format was with the old `.md`, wonder if samething with new `.rst`.

Reply:
I'm testing things with symlinks and relpaths, maybe this approach isn't working.